hasher-matcher-actioner/webapp/src/pages/Matches.jsx

```diff
@@ -118,54 +132,29 @@ function MatchListFilters({searchInAttribute, searchQuery}) {
   const inputRef = React.createRef();
 
   return (
-    <Form onSubmit={loadSearchPage}>
-      <Form.Group>
-        <Row className="mt-1">
+    <Form className="mx-4" onSubmit={loadSearchPage}>
+      <Form.Group className="mb-0">
+        <Row className="my-2">
           <InputGroup>
+            <InputGroup.Prepend>
+              <Form.Control
+                as="select"
+                value={localSearchInAttribute}
+                onChange={e => setLocalSearchInAttribute(e.target.value)}>
+                <option value="contentId">Content ID</option>
+                <option value="signalId">Signal ID</option>
+              </Form.Control>
+            </InputGroup.Prepend>
             <FormControl
               ref={inputRef}
               onChange={e => setLocalSearchQuery(e.target.value)}
               value={localSearchQuery || ''}
-              htmlSize="40"
             />
             <InputGroup.Append>
-              <Button onClick={loadSearchPage}>Search</Button>
+              <Button onClick={loadSearchPage}>Filter</Button>
```

Comment:
The word 'Search' is all over this file...
I agree the Filter is better but I think we should also refactor `setLocalSearchInAttribute` etc., while we are here to things like `setLocalFilterInAttribute` (to avoid the code drifting from the user experience/docs)

Reply:
Makes sense. Will do.